docs/tutorials/projects/light-up.md

```diff
@@ -84,12 +79,83 @@ Navigate to the **Control** tab where you can see your camera working.
 
 {{< gif webm_src="../../img/light-up/control-panel.webm" mp4_src="../../img/light-up/control-panel.mp4" alt= "Opening the camera panel on the control tab, toggling the video feed on, and watching as a person wearing headphones waves at the camera." >}}
 
+## Configure your services
+
+This tutorial uses pre-trained ML packages.
+If you want to train your own, you can [train a model](/manage/ml/train-model/).
+
+To use the provided Machine Learning model, copy the <file>[effdet0.tflite](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/effdet0.tflite)</file> file and the <file>[labels.txt](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/labels.txt)</file> to your project directory.
+
+{{< tabs >}}
+{{% tab name="Builder UI" %}}
```

Comment:
The easiest thing is to remove these tabs. 
```suggestion
```

They are in the tipsy tutorial because I also show how you can configure it with raw json. Alternatively you can copy that part as well. 

The markup for tabs is:
```
{{< tabs >}}
{{% tab name="Builder UI" %}}

{{% /tab %}}
{{% tab name="Raw JSON" %}}

{{% /tab %}}
{{< /tabs >}}
```